client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -32,12 +32,10 @@ class TwoFactorActions extends Component {
 	sendSmsCode = ( event ) => {
 		event.preventDefault();
 
-		const { userId, twoStepNonce, translate } = this.props;
-
-		page( login( { twoFactorAuthType: 'sms' } ) );
+		const { userId, twoStepNonce } = this.props;
 
 		this.props.sendSmsCode( userId, twoStepNonce ).then( () => {
-			this.props.successNotice( translate( 'Recovery code has been sent.' ) );
+			page( login( { twoFactorAuthType: 'sms' } ) );
```

Comment:
I think this change is a mistake. What if I send the SMS and then switch to an authenticator app, and then want to switch back to SMS? I will no longer be able to, as we can't send another SMS.